profiler/src/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(PROFILER_SOURCES
     profile_conv_tensor_rearrange.cpp
     profile_transpose.cpp
     profile_permute_scale.cpp
+    profile_gemm_universal.cpp
```

Comment:
We already have profile target in Line 6 of this file

Reply:
done